data/sql/db-world/trasm_world_texts.sql

```diff
@@ -51,4 +51,4 @@ INSERT INTO `command` (`name`, `security`, `help`) VALUES
 ('transmog add', 1, 'Syntax: .transmog add $player $item\nAdds an item to a player\'s appearance collection.'),
 ('transmog sync', 0, 'Syntax: .transmog sync\nSyncs transmog addon appearances with the server.'),
 ('transmog add set', 1, 'Syntax: .transmog add set $player $itemSet\nAdds items of an item set to a player\'s appearance collection.'),
-('transmog portable', 1, 'Syntax: .transmog portable \nSummons the Ethereal Warpweaver, a portable version of the transmogrification NPC.');
+('transmog portable', 0, 'Syntax: .transmog portable \nSummons the Ethereal Warpweaver, a portable version of the transmogrification NPC.');
```

Comment:
Keep this at 1, it should be changed at the end user discretion

Reply:
if I put this 1 I can't enable it for transmog plus, moreover, if I put it 0 the players can't anyway use it if they do not enable transmog plus or they have security level moderator

if you think that is a risk to leave it as 0 I can rollback as 1

Reply:
This can be changed by any user at their discretion, including Chromiecraft, so we can change it as we need it at our end